scrapy/utils/project.py

```diff
@@ -1,6 +1,7 @@
 import os
 from six.moves import cPickle as pickle
 import warnings
+from scrapy.exceptions import ScrapyDeprecationWarning
```

Comment:
one more nitpick: could you please move this import to another scrapy.exceptions import below? According to pep8, imports should be grouped in 3 groups: stdlib imports, third-party imports, project imports; this one should be in  the last group.

Reply:
@kmike  missed that one thanks for the suggestion. I have made the changes accordingly.